### PR TITLE
[#IOCIT-174] APIM Policy for progressive rollout fn-services v4

### PR DESCRIPTION
**Pull request by AleDore**
### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [ ] DEV
- [ ] UAT
- [ ] PROD

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### How to apply

After PR is approved
1. run deploy pipeline from Azure DevOps [io-platform-iac-projects](https://dev.azure.com/pagopaspa/io-platform-iac-projects/_build?view=folders)
2. select PR branch
3. wait for approval

### CODE REVIEW

**Review by pasqualedevita — APPROVED** [review]

File: src/core/api/io_services/v1/policy.xml [hunk 1]
```diff
@@ -16,29 +16,37 @@
                 <set-backend-service base-url="https://io-p-services-fn-2.azurewebsites.net/api/v1" />
             </when>
             <otherwise>
-                <set-backend-service id="apim-generated-policy" base-url="{{io-fn3-services-url}}/api/v1" />
                 <!-- https://docs.microsoft.com/en-us/dotnet/api/system.random.next?view=net-6.0#system-random-next(system-int32-system-int32) -->
-                <!-- <set-variable name="urlWeight" value="@{
+                <set-variable name="urlWeight" value="@{
```
Comment (balanza) [inline src/core/api/io_services/v1/policy.xml:20]: To help reading the policy, can you please add a block comment in plain english to explain the intentions of the following code? Readers must go though implementation details and reverse-engineer the whole algorithm to understand it.
